lib/shipcloud/shipment.rb

```diff
@@ -9,7 +9,8 @@ class Shipment < Base
     attr_accessor :from, :to, :carrier, :service, :package, :reference_number, :metadata,
                   :additional_services
     attr_reader :id, :created_at, :carrier_tracking_no, :tracking_url, :label_url,
-                :packages, :price, :customs_declaration, :pickup, :label_voucher_url
+                :packages, :price, :customs_declaration, :pickup,
+                :label_voucher_url, :carrier_tracking_url
```

Comment:
There's a corresponding spec file at `spec/shipcloud/shipment_spec.rb`, could you add a spec for this new attribute?